src/ec/packages/ec-component-page-header-standardised/page-header-standardised.story.js

```diff
@@ -9,17 +11,46 @@ import demoMetaTitleDescriptionContent from './demo/data--meta-title-description
 import pageHeaderStandardised from './ecl-page-header-standardised.html.twig';
 import notes from './README.md';
 
-demoTitleContent.breadcrumb.icon_file_path = defaultSprite;
-demoMetaTitleContent.breadcrumb.icon_file_path = defaultSprite;
-demoMetaTitleDescriptionContent.breadcrumb.icon_file_path = defaultSprite;
+// Labels for the groups.
+const requiredGroupId = 'Mandatory elements';
+const optionalGroupId = 'Optional elements';
+
+const preparePageHeaderStandardised = data => {
+  data.breadcrumb.icon_file_path = defaultSprite;
+  data.title = text('title', data.title, requiredGroupId);
+  data.breadcrumb = object('breadcrumb', data.breadcrumb, requiredGroupId);
+
+  if (data.meta) {
+    data.meta = text('meta', data.meta, optionalGroupId);
+  }
+  if (data.description) {
+    data.description = text('description', data.description, optionalGroupId);
+  }
+
+  data.extra_classes = text(
+    'extra_classes (comma separated)',
+    '',
+    optionalGroupId
+  );
+  data.extra_attributes = object(
+    'extra_attributes',
+    { name: '', value: '' },
+    optionalGroupId
+  );
```

Comment:
This doesn't work in the when in check the html

Reply:
i'll check it